usr/local/www/status_gateway_groups.php

```diff
@@ -54,113 +54,117 @@
 $shortcut_section = "gateway-groups";
 include("head.inc");
 
+$tab_array = array();
+$tab_array[0] = array(gettext("Gateways"), false, "status_gateways.php");
+$tab_array[1] = array(gettext("Gateway Groups"), true, "status_gateway_groups.php");
+display_top_tabs($tab_array);
 ?>
 
-<body link="#0000CC" vlink="#0000CC" alink="#0000CC">
-<?php include("fbegin.inc"); ?>
-	<table width="100%" border="0" cellpadding="0" cellspacing="0">
+<div class="table-responsive" id="mainarea">
+	<table class="table table-hover table-striped table-condensed">
 		<tr>
-		  <td>
+			<th class="col-md-1"><?=gettext("Group Name"); ?></th>
```

Comment:
What's the reason you added column size-classes here? I'm pretty sure they're not needed